woocommerce/woocommerce-android
# [Mobile Payments] JITMs action should be able to handle universal links

Closes: #8460


### Description

The PR  makes JITM more versatile, so it now handles the given link as a universal link. So if link is supported by the app then specific screen in the app is open, if not then browser is open

### Testing instructions

* Receive JITM IPP
* Click on "purchase card reader"
* Notice that Web view is opened

Do the same but alter the URL returned from the [JITM endpoint](path=%2Fjetpack%2Fv4%2Fjitm%2F%26_method%3Dget&json=true&query=%7B%22message_path%22%3A%22woomobile%3Amy_store%3A) to `https://woocommerce.com/mobile/payments/tap-to-pay`  using code change or proxy change the URL. Notice that the "TTP summary screen" is opened

### Images/gif
